### PR TITLE
Review H letter icons

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
## Review of all icons starting with letter `H`

**Issue:** Related to #9072 

1. Check commits for list of fixed icons.
2. Issues are discussed under comments on this PR

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5717,7 +5718,7 @@
         {
             "title": "Hackaday",
             "hex": "1A1A1A",
-            "source": "https://hackaday.com"
+            "source": "https://hackaday.com/wp-content/themes/hackaday-2/img/logo.svg"
```
Comment (LitoMore) [inline _data/simple-icons.json:5721]: I prefer to keep the domain only. The link to the WordPress theme asset is subject to change.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -5706,7 +5706,8 @@
         {
             "title": "Hack Club",
             "hex": "EC3750",
-            "source": "https://hackclub.com/brand"
+            "source": "https://hackclub.com/brand",
+            "guidelines": "https://hackclub.com/brand/"
```
Comment (LitoMore) [inline _data/simple-icons.json:5710]: Trailing slash.

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json:5721]: The SVG version doesn't exist on the front page. So in that case should we still link to front page?

Comment (LitoMore) [reply _data/simple-icons.json:5721]: Please let us know how you find this resource link.

**Comment by PratyushJayachandran** [comment]
Issues with icons:

1. HBO: https://www.hbo.com/terms-of-use makes it seem like the logo cannot be used
2. HelloFresh: Logo has changed https://www.hellofresh.com/about/press#question-US-Mediaassets-Logo-0. Should we change it?
3. Hilton: The source link is leading to a "potential security risk" page. Regardless, the icon used is not representative of the Hilton brand as given here: https://stories.hilton.com/media-kit-hilton-corporate. What do we do?
4. Httpie: How did we pick up this color `#73DC8C`? https://github.com/httpie/cli/blob/master/docs/httpie-logo.svg this shows a different color.

@simple-icons/icon-review Please take a look at these.

**Review by adamrusted — COMMENTED** [review]
In theory, Harbor and Hive.io should be in a separate PR that we can 'skip' in terms of just removing the slashes. I'm not _super_ worried about that though, given it's only 2 icons!

**Comment by adamrusted** [comment]
> * HBO: https://www.hbo.com/terms-of-use makes it seem like the logo cannot be used

Added in 2020, before we were quite as stringent on terms of use adherence.
Wikimedia suggests we can still include it, unless HBO come after us.
image


> * HelloFresh: Logo has changed https://www.hellofresh.com/about/press#question-US-Mediaassets-Logo-0. Should we change it?

There's already #5559 open to discuss this :)

> * Hilton: The source link is leading to a "potential security risk" page. Regardless, the icon used is not representative of the Hilton brand as given here: https://stories.hilton.com/media-kit-hilton-corporate. What do we do?

It's because their redirect service from the old domain isn't HTTPS - no worries, the destination _is_ secure.
We'll need to update the Hilton icon - if you want to open a separate issue to track that?

> * Httpie: How did we pick up this color `#73DC8C`? https://github.com/httpie/cli/blob/master/docs/httpie-logo.svg this shows a different color.

See https://github.com/simple-icons/simple-icons/pull/7331#pullrequestreview-948650324

File: _data/simple-icons.json (hunk 1 above)
Comment (adamrusted) [reply _data/simple-icons.json:5721]: It _is_ available in the header of the site as the 'mask icon' link. I'd say let's just keep the homepage URL.

**Comment by adamrusted** [comment]
@PratyushJayachandran - now you're a maintainer, you can edit my icon review post too, if you fancy leaving an update on 'H' there? 😇

**Comment by PratyushJayachandran** [comment]
Issues continued:
5. Hurriyetemlak has changed hands I guess. This seems to be the new website: https://www.hepsiemlak.com/en. Should we remove or update?

**Comment by PratyushJayachandran** [comment]
Icon updates will be handled in separate PRs.
All other minor corrections are complete on this PR.

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json]: Reverted this change in another commit

**Comment by adamrusted** [comment]
> 5. Hurriyetemlak has changed hands I guess. This seems to be the new website: https://www.hepsiemlak.com/en. Should we remove or update?

Two approaches here, either create a PR to replace the icon entirely - including rename and update. OR create one to add 'Hepsiemlak' - and another to remove 'Hurriyetemlak' as of v10.

> 6. Hyperledger cannot be used I think: https://www.hyperledger.org/trademark-usage

If you're not happy with the trademark terms, feel free to open a v10 PR for removal. We can always discuss when it comes time to review those PRs!

**Review by adamrusted — APPROVED** [review]

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - now you're a maintainer, you can edit my icon review post too, if you fancy leaving an update on 'H' there? 😇
> 

I accidentally edited he message instead of replying :P

> I updated once and got scared with that wall of text 😨

**Pull request merged**